src/test/java/cucumber/steps/SetTagBodyStep.java

```diff
@@ -26,9 +25,11 @@ public SetTagBodyStep(ApiRequestBuilder apiRequestBuilder, ApiResponse apiRespon
 
     @When("I set the tags body with following values:")
     public void setTagBody(final Map<String, String> body) {
-        List<String> trashList =  new ArrayList<>();
-        trashList.add(body.get("name"));
-        scenarioContext.setTrash("Tags Trash", trashList);
+        LinkedList<String> trashList =  new LinkedList<>();
+        if (body.get("name") != null) {
```

Comment:
is it ok that the name is hardcoded? It will always use that parameter?

Reply:
Yes it is necessary, because Tags always uses name instead of an ID